fmn/api/main.py

```diff
@@ -109,9 +109,8 @@ async def get_user_rules(
     if username != identity.name:
         raise HTTPException(status_code=403, detail="Not allowed to see someone else's rules")
 
-    return (
-        await db_session.execute(Rule.select_related().filter(Rule.user.has(name=username)))
-    ).scalars()
+    db_result = await db_session.execute(Rule.select_related().filter(Rule.user.has(name=username)))
+    return [Rule.from_orm(rule) for rule in db_result.scalars()]
```

Comment:
We shouldn't have to manually convert to API model objects, `response_model=` above should take care of it. And plain `Rule` is the ORM class, not the API model class.